addons-cluster/gbase-cluster/templates/ssh_secret_generate.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
```

Comment:
maybe we can render ssh keys in a go tpl fun , to simplify the procedure. (such that all pods of the same cmp will share the same keys, and other pods for backup/restore, can mount ssh keys, will discuss with @sophon-zt